kolibri/core/assets/src/api-resources/contentNode.js

```diff
@@ -48,6 +48,37 @@ export default class ContentNodeResource extends Resource {
     }
     return promise;
   }
+
+  getCopies(id) {
```

Comment:
Maybe just change this `id` to `content_id` to make it clear we are fetching by a content_id and not a pk?